src/components/screen/Calender/Calender.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  format,
+  getDate,
+  getDay,
+  addMonths,
+  subMonths,
+  getMonth,
+  endOfWeek,
+  eachDayOfInterval,
+  eachWeekOfInterval,
+  startOfMonth,
+  endOfMonth,
+} from 'date-fns'
+import { useState } from 'react'
+import { CalenderProps } from './Calender.types'
+
+const Calendar: React.FC<CalenderProps> = (onChange) => {
```

Comment:
今後propsを増やすことも考えて、objectの要素の一つとして受け取っちゃっていいかな

```suggestion
const Calendar: React.FC<CalenderProps> = ({ onChange }) => {
```

Reply:
修正した
https://github.com/NUTFes/chosei-chan/pull/22/commits/4c143888c3fe8faa81b540ed04bcd36b8dfac5c1